jni/src/test/java/com/horizen/schnorrnative/SchnorrPublicKeyTest.java

```diff
@@ -0,0 +1,22 @@
+package com.horizen.schnorrnative;
+
+import com.horizen.librustsidechains.FieldElement;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class SchnorrPublicKeyTest {
+
+    @Test
+    public void testGetHash() throws Exception {
+        byte[] seed = { 1, 2, 3, 4, 5, 6, 7, 8 };
+
+        try(SchnorrKeyPair keyPair = SchnorrKeyPair.generate(seed))
+        {
+            assertNotNull("Key pair generation was unsuccessful.", keyPair);
+            FieldElement hash = keyPair.getPublicKey().getHash();
+            byte[] hashBytes = hash.serializeFieldElement();
+            System.out.println("hash bytes: " + hashBytes);
```

Comment:
Let's turn this into a regression test.:Hardcode the result and check the output

Reply:
Ha yes that’s the plan I wasn’t finished yet

Reply:
Done